apps/api/src/persistence/migrations/coscrad-query-runner.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { HasId } from '@coscrad/api-interfaces';
+import { DatabaseDTO } from '../database/utilities/mapEntityDTOToDatabaseDTO';
+
+export interface UpdateQueryOptions<TOldDocument> {
+    propertiesToRemove?: (keyof TOldDocument)[];
+}
+
+/**
+ * There's not much reason for this interface if we are going to use `DatabaseDTO`,
```

Comment:
Let's do this for now. We can always generalize later if we at some point need a second persistence option.

Reply:
I decided to leave a comment for now.